react-frontend/src/utils/apolloClient.js

```diff
@@ -0,0 +1,8 @@
+import ApolloClient from 'apollo-boost';
+
+const client = new ApolloClient({
+  // uri: `${process.env.STRAPI_BACKEND_URL}/graphql` || "http://localhost:1337/graphql",
```

Comment:
commented out code/ why is the env var not used?

Reply:
i suspect it can be leveraged using `REACT_APP_STRAPI_BACKEND_URL` ?

Reply:
There is some kind of config issue keeping the environment variables from being populated.  Working on a fix right now :P but I didn't want to block the rest of the team.

Reply:
client side env vars need to be prefiixed with `REACT_APP` could that be it?